### DISCUSSION TITLE
Does a v3 fork exist?

### DISCUSSION THREAD

**Discussion by AppleSheeple** [transmission/transmission#6968] [General]
Hello.

I hope this doesn't sound like I'm trolling, but is anyone maintaining/updating v3 in a fork?

I understood the C code better, and had a few minor patches applied. So, I've been avoiding the v4 rewrite.

I only use daemon/remote.

**Comment by killemov** [comment]
AFAIK No. Both v3 and v4 still have issues at this time of writing but the v4 ones are getting fixed and the v3 ones are not. So pick your moment of upgrading to v4 based on features/performance and rewrite and reapply your patches.

**Comment by AppleSheeple** [comment]
While doing some searches related to a couple of build issues I had after updating my system, I bumped into a fork which looks legit:
https://github.com/stefantalpalaru/transmission-og

Just a couple of notes for anyone interested:
1. git submodules are used for third_party libs, and one has to explicitly pass `--enable-external-libfoo` for any library where the system version is preferred.
2. I had to pass ` --without-qt` explicitly to disable building that (already had `--without-gtk` passed).
3. Executables depend on a shared `libtransmission-og.so`, so remember to install it.
4. ~~Everything is renamed with the `-og` suffix. That includes configuration folder(s). So don't panic if you see none your previous session data. You just need to copy/link the configuration folder(s) first to the new path(s).~~ no longer needed in v3.04

Thanks you @stefantalpalaru for doing this work.

**Comment by stefantalpalaru** [comment]
> Everything is renamed with the -og suffix. That includes configuration folder(s).

Configuration folders are not supposed to be renamed. Please submit a bug report.
